CollectSamError/CollectSamErrorMetrics.wdl

```diff
@@ -43,15 +43,27 @@ task CollectSamErrorMetricsTask {
         File intervals
         Int disk_gb = 400
         Int memory_gb = 16
+        Int minMQ = 60
+
     }
     command {
         java -Xms2000m -jar /usr/gitc/picard.jar CollectSamErrorMetrics I=~{input_bam} \
-        O=~{output_name} R=~{ref_fasta} V=~{vcf_file} L=~{intervals} \
+        O=~{output_name} R=~{ref_fasta} V=~{vcf_file} L=~{intervals} MQ=~{minMQ} \
         ERROR_METRICS=null \
         ERROR_METRICS="ERROR:INSERT_LENGTH" \
         ERROR_METRICS="ERROR:CYCLE" \
         ERROR_METRICS="ERROR:GC_CONTENT" \
-        ERROR_METRICS="ERROR"
+        ERROR_METRICS="ERROR" \
+        ERROR_METRICS="ERROR:CYCLE:BASE_QUALITY:ONE_BASE_PADDED_CONTEXT" \
```

Comment:
Let's remove this context:
ERROR_METRICS="ERROR:CYCLE:BASE_QUALITY:ONE_BASE_PADDED_CONTEX